static/templates/settings/organization-settings-admin.handlebars

```diff
@@ -108,5 +109,21 @@
     <div class="input-group organization-submission">
       <input type="submit" class="button" value="{{t 'Save changes' }}" />
     </div>
+    </div>
+    <div class="realm-icon-section box-shadow border-radius">
+      <div class="inline-block">
+        <img id="realm-settings-icon" src="{{ realm_icon }}"/>
+        <div id="realm_icon_file_input_error" class="text-error"></div>
+        <input type="file" name="realm_icon_file_input" class="notvisible"
+               id="realm_icon_file_input" value="{{t 'Upload icon' }}"/>
+        <div id="upload_icon_spinner"></div>
+      </div>
+      <div class="inline-block">
+        <button class="button sea-green w-200 m-t-10 block input-size"
+                id="realm_icon_upload_button">{{t 'Upload New Icon' }}</button>
+        <button class="button btn-danger w-200 m-t-10 block input-size"
+                id="realm_icon_delete_button">{{t 'Delete Icon' }}</button>
```

Comment:
For spelling this, we always do e.g. "Upload new icon", with just the first letter capitalized.